packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -281,8 +294,9 @@ class _DropdownRouteResult<T> {
 
   @override
   bool operator ==(dynamic other) {
-    if (other is! _DropdownRouteResult<T>)
+    if (other is! _DropdownRouteResult<T>) {
```

Comment:
Previous syntax was fine: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#separate-the-if-expression-from-its-statement